.github/ISSUE_TEMPLATE/000-incubation-review.yaml

```
@@ -0,0 +1,73 @@
+name: Early design/incubation review
+description: >
+  If you'd like the TAG to offer thoughts and guidance
+  on an early-stage design direction.
+title: "Incubation: "
+labels:
+  - "Progress: untriaged"
+  - "Review type: CG early review"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for sending us your design review.
+
+        Use links to content rather than pasting text into this issue. Issues are ephemeral and most of the material we are asking for has long term value.
+
+        Please preview the issue and check that the links work before submitting. Please make sure anyone with the link can access the document. We may refuse to review anything that is not public.
+
+  - type: input
+    attributes:
+      label: Explainer
+      description: An explainer must address user needs and contain examples of use. See our [explanation of how to write a good explainer](https://w3ctag.github.io/explainer-explainer/#introduction).
```

My only concern with the examples of usage is that it's asking for a solution to be "engineered". That risks locking in some kind of design (e.g., JS vs markup or whatever) - where there might be multiple ways to solve a problem. 

```suggestion
      description: An explainer must explain the value to end users and contain examples of use, even if hypothetical. See our [explanation of how to write a good explainer](https://w3ctag.github.io/explainer-explainer/#introduction).
```